common/extensions/api/brave_rewards.json

```diff
@@ -379,6 +379,24 @@
       }
     ],
     "functions": [
+      {
```

Comment:
if this is a utility function that can be used from any internal extension it should go in it's own api config instead of being duplicated in rewards and shields

Reply:
as discussed, it's specific to each API (each API opens the UI related to that API)